src/App.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import TaskBox from "./components/TaskBox";
+
+import BeadsLogo from "./logos/beads.svg";
+import MsitLogo from "./logos/msit.svg";
+import ProvocationLogo from "./logos/provocation.svg"
+
+function App() {
+  let query = new URLSearchParams(window.location.search);
+  const participantId = query.get("participantID");
+  const studyId = query.get("studyID");
```

Comment:
is there a way to more literally pass through all of the query params when you construct the links below? that way if we decide to add some other query param we're future proof?